qemu/qemu_test.go

```diff
@@ -236,3 +236,25 @@ func TestAppendQMPSocket(t *testing.T) {
 
 	testAppend(qmp, qmpSocketString, t)
 }
+
+var qemuString = "-name cc-qemu -cpu host -uuid 123456789"
+
+func TestAppendStrings(t *testing.T) {
+	var params []string
+
+	config := Config{
+		Path:     "qemu",
+		Name:     "cc-qemu",
+		UUID:     "123456789",
```

Comment:
The unit test is about testing that the generated string is valid. Validating the actual input is an orthogonal and more general issue and also out of this patch scope.
I can use a valid UUID for this test, but the overall input validation strategy needs to be defined at a higher level and for the project as a whole, not as a specific component or even patchset level.
In that specific case, the qemu parameters are derived from user input but not used directly as is. Can we do qemu parameter injection ? Maybe, but this needs to be audited and then we can decide *where* we do input validation. Doing that at the end of the code path, which is where this code lives, is probably not the right thing to do.